docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,38 @@
+# Network Server IoT Edge Module configuration
+
+The following table is providing a list of configuration options, to be provided as environment variables for manual configuration of the `LoRaWanNetworkSrvModule`:
+
+| Environment variable name   | Description                                                                            | Mandatory                           |
+| --------------------------- | -------------------------------------------------------------------------------------- | ----------------------------------- |
+| IOTEDGE_APIVERSION          | Version of IoT Edge; also used to determine if LNS is running as an IoT Edge module    | Yes, if running as IoT Edge module  |
+| IOTEDGE_IOTHUBHOSTNAME      | Host name of the IoT Hub                                                               | Yes                                 |
+| IOTEDGE_GATEWAYHOSTNAME     | Gateway host name                                                                      | Yes, if edgeHub gateway is enabled  |
+| ENABLE_GATEWAY              | Indicates whether the edgeHub gateway should be enabled or not                         | No (defaults to `true`)             |
+| IOTEDGE_DEVICEID            | The gateway deviceId                                                                   | Yes                                 |
+| HTTPS_PROXY                 | HTTPS proxy url                                                                        | No                                  |
+| RX2_DATR                    | RX2 data rate                                                                          | No (defaults to 1)                  |
+| RX2_FREQ                    | RX2 frequency                                                                          | No                                  |
```

Comment:
Useful to override the default regional RX2 Frequency at a global level (defaults to null, meaning that we take the regional values)